### DISCUSSION TITLE
Amazon Customer Review Request Feature

### DISCUSSION THREAD

**Discussion by briansackett** [Sellbrite/Sellbrite-API#10] [Q&A]
Hello,

Is there a feature available for triggering the amazon request a review? It looks like the API exists for 3P platforms, I think the utility of it would warrant the development.

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

We currently don’t have a feature that would automate or trigger the Amazon request for a review process. I will be happy to submit this as a feature request to our product team on your behalf. Keep the feedback coming! We love hearing what our customers have to say and take customer requests into consideration as we determine future developments.
